integration/integration-1/integration-1Deployment.yaml

```diff
@@ -21,7 +21,7 @@ datarepo-api:
   image:
     tag: 68ec114-develop
   env:
-    GOOGLE_PROJECTID: broad-jade-int-1
+    GOOGLE_PROJECTID: broad-jade-integration
```

Comment:
are we going to drop the broad-jade-int-1 and broad-jade-int-2 projects completely?

Reply:
I think so, yes.